src/sql/workbench/contrib/notebook/browser/cellViews/textCell.component.ts

```diff
@@ -232,7 +244,7 @@ export class TextCellComponent extends CellView implements OnInit, OnChanges {
 		this.toggleEditMode(this.isActive());
 
 		if (this.output && this.output.nativeElement) {
-			(<HTMLElement>this.output.nativeElement).scrollTo({ behavior: 'smooth' });
+			(<HTMLElement>this.output.nativeElement).scrollIntoView({ behavior: 'smooth' });
```

Comment:
how does this behavior change?

Reply:
scrollTo haven't worked so I updated it to scrollIntoView.

Reply:
It didn't scroll at all, so I updated it.